kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/Tab8021xUi.java

```diff
@@ -348,6 +352,14 @@ private void initPrivateKeyNameTextBox() {
         });
     }
 
+    private void update() {
+        logger.info("update");
+        setValues();
+        refreshForm();
+        this.netTabs.updateTabs();
+        logger.info("update... done");
```

Comment:
Only thing here is... is this log entry worth the `info` level or would it be better suited for the `debug` level?

```suggestion
        logger.debug("update");
        setValues();
        refreshForm();
        this.netTabs.updateTabs();
        logger.debug("update... done");
```